src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return basePrice - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM) {
+    return basePrice - SHORT_TERM_DISCOUNT;
+  }
+
+  return basePrice;
```

Comment:
The function `calculateRentalCost` is well-named and describes its purpose effectively. Additionally, the use of constants instead of hardcoded literals for `PRICE_PER_DAY`, `LONG_TERM`, `LONG_term_DISCOUNT`, `SHORT_TERM`, and `SHORT_TERM_DISCOUNT` is good as it makes the code more readable and maintainable. Moreover, the use of `const` for these values is appropriate since they should not change. The structure of the `if` statements also aligns with best practices by returning early and not using `else` after an `if` that contains a `return` statement.